vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -252,6 +254,19 @@ public String recommendationsUri() {
         return recommendationsUri;
     }
 
+    /**
+     * The password for the video, only sent when the following conditions are true
+     * 1. The privacy is set to password
+     * 2. The user making the request owns the video
+     * 3. The application making the request is granted access to view this field
```

Comment:
I think this _might_ show up as one line. If you have it open in your IDE, I would mouse over getPassword and just check if the JavaDoc looks good. 

For numbered lists it might be preferable to use <ul> <ol></ol> </ul>. But if this looks good in the JavaDoc, then it's totally fine.